code/modules/reagents/reagent_containers/hypospray.dm

```diff
@@ -79,6 +79,7 @@
 	icon_state = "medivend_hypo_bluespace"
 	item_state = "medivend_hypo_bluespace"
 	volume = 100
+	ignore_flags = 1

```

Comment:
Might want to use `TRUE` here

Reply:
Again just trying to use the conventions of previous items, the combat hypo used 1 so I used one.

Reply:
If it's a true or false value then you'll want to use true or false. Someone's going to go through the code and eventually change those so it would help to make them consistent with the standard and not the conventions